front/app/modules/commercial/project_folders/citizen/components/ProjectFolderGoBackButton/index.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen, fireEvent } from 'utils/testUtils/rtl';
+
+import GoBackButton from './';
+
+const projectFolderData = {
+  id: 'a8735d6f-cdd4-4242-8531-b089b5e3954a',
+  type: 'folder',
+  attributes: {
+    title_multiloc: {
+      en: 'TestFolder',
+      'fr-BE': 'Le folder du test',
+      'nl-BE': 'TestMap',
+    },
+    slug: 'testfolder',
+  },
+};
+
+let historyState: string;
+const history = {
+  push(location: string) {
+    historyState = location;
+  },
+};
+
+jest.mock('utils/cl-router/history', () => history);
+jest.mock('../../../hooks/useProjectFolder', () =>
+  jest.fn(() => projectFolderData)
+);
+jest.mock('hooks/useLocale', () => jest.fn(() => 'en'));
```

Comment:
We already have global mocks for useLocale and useLocallize, so maybe you can use them?
Just adding the following should do the trick :) 
```
jest.mock('hooks/useLocale')
jest.mock('hooks/useLocalize')
```